internal/service/users_v1/ocm_interface.go

```diff
@@ -0,0 +1,47 @@
+package users
+
+import (
+	"context"
+	"os"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/openshift-online/ocm-sdk-go/logging"
+)
+
+type OcmSDK struct{}
```

Comment:
The idea of having an interface here is great :ok_hand:, just need the actual interface created. 

---

Are you going to create a mocked out version for testing? If not that ok I can make one too.

Reply:
I certainly can!!  I'll make one and push it up to this PR